app/assets/javascripts/koi/controllers/index_actions_controller.js

```diff
@@ -26,6 +26,9 @@ export default class IndexActionsController extends Controller {
   }
 
   update() {
+    // Turbo permanent doesn't allow the server to set the value attribute on reload
+    // Manually setting the value here for test support
+    this.searchTarget.setAttribute("value", this.searchTarget.value);
```

Comment:
in Tables I changed the test to read the value from JS:

```
expect(find("input[type=search]").value).to eq("1")
```

I think it's better to change the test vs change the code

Reply:
Thanks for the feedback, agreed and changed the spec to match the new code